front/src/utils/share/share.util.test.ts

```diff
@@ -0,0 +1,42 @@
+import { ShareUtils } from "..";
+
+describe("Share utils", () => {
+  const title = "Test title";
+  const message = "Test message";
+  const url = "https://testurl.com";
+
+  describe("buildShareContent", () => {
+    it("buildShareContent is called with url on Android", () => {
```

Comment:
`should return share content with message and url concatenation when url is defined on Android`